src/pages/GroupDetail.tsx

```
@@ -509,16 +509,37 @@ export default function GroupDetail() {
-              <img
-                src={image}
-                alt={name}
-                className="w-full h-full object-cover object-center"
-                onLoad={() => setImageLoading(false)}
-                onError={(e) => {
-                  setImageLoading(false);
-                  e.currentTarget.src = "/placeholder-community.svg";
-                }}
-              />
+              {/* Check if the image URL is likely a video */}
+              {image && image.match(/\.(mp4|webm|ogg|mov)$/i) ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)